lib/analytics/index.js

```diff
@@ -10,14 +10,20 @@ const analytics = {
     analytics.identifyUser();
   },
 
+  // Return a valid platform prefix when on a platform that should be tracked
   getPlatformPrefix: function() {
-    if (navigator.appVersion.indexOf('Win') !== -1) {
-      return 'spwindows_';
-    } else if (navigator.appVersion.indexOf('Linux') !== -1) {
-      return 'splinux_';
+    if (!navigator.appVersion.includes('Electron')) {
+      return 'spweb';
     }
 
-    return 'unknown_';
+    switch (true) {
```

Comment:
Is this better than an `if` statement here? 🤔

Reply:
My unconscious mind was wishing for 'Mac' to join my list of valid platforms 😁